api/ClipboardItem.json

```diff
@@ -124,6 +130,100 @@
           }
         }
       },
+      "createDelayed": {
```

Comment:
I'd drop the new features in this PR. None of the three new features are listed as supported by any browser. Unless we have evidence that implementation is imminent, I think it's too soon to add it to BCD. After all, there's no compat risk if something has never been supported by any browser.